pkg/api/types.go

```diff
@@ -1064,12 +1064,10 @@ type NodeConditionType string
 
 // These are valid conditions of node. Currently, we don't have enough information to decide
 // node condition. In the future, we will add more. The proposed set of conditions are:
-// NodeReachable, NodeLive, NodeReady, NodeSchedulable, NodeRunnable.
+// NodeReady
```

Comment:
I think we plan to add NodeReachable back in the future maybe, so I would add that one to the list.